File: spring2013/codeWar/LRS/Clients/PlayerCppAI/Robot.cpp
#include "Robot.h"

namespace PlayerCSharpAI
{
	namespace api
	{
		Robot::Robot() : CombatUnit()
		{

		}

		Robot::Robot(const xml_node& elemOn) : CombatUnit(elemOn)
		{

		}

		Robot::Robot(const Robot& r) : CombatUnit(r)
		{

		}

		Robot& Robot::operator=(const Robot& r)
		{
			CombatUnit::operator=(r);
			return *this;
		}
	}
}

File: spring2013/codeWar/LRS/Clients/PlayerCSharpAI/api/Robot.cs
// Created by Windward Studios, Inc. (www.windward.net). No copyright claimed - do anything you want with this code.

using System.Xml.Linq;

namespace PlayerCSharpAI.api
{
	/// <summary>
	/// A player's robot.
	/// </summary>
	public class Robot : CombatUnit
	{
		/// <summary>
		/// Build from XML.
		/// </summary>
		/// <param name="elemOn">Initialize with the values in this object.</param>
		public Robot(XElement elemOn) : base(elemOn)
		{
		}
	}
}
